src/features/Faucet/helpers.tsx

```diff
@@ -76,12 +60,7 @@ export const makeTxConfig = (
    * getFaucetContact() returns ExtendedContact, which is the closest we can get.
    * The result is casted to `any` to make it compatible with ITxConfig.
    */
-  const senderContact: any = getFaucetContact(
-    txResult,
-    networks,
-    getContactByAddressAndNetworkId,
-    createContact
-  );
+  const senderContact: any = getFaucetContact(txResult, networks, getContactByAddressAndNetworkId);
```

Comment:
Replace any with a more strict type if possible

Reply:
I changed it to a cast